AppInspector/rules/default/general/dependencies.json

```diff
@@ -22,14 +20,12 @@
     "name": "Dependency: Included Source",
     "id": "AI026500",
     "description": "Dependency: Included Source",
-    "tags": [
-      "Dependency.SourceInclude"
-    ],
+    "tags":[ "Dependency.SourceInclude" ],
     "applies_to": [ "php" ],
     "severity": "moderate",
     "patterns": [
       {
-        "pattern": "(include|require) .*;",
+        "pattern": "(include|require) [A-Za-z];",
```

Comment:
You probably want to expand this so when the pattern is captured it's readable - Maybe instead of `.*`, we use `[^\s;]{1,30};` or something like that?